### PR TITLE
Added Hex VTK output

### PR DESCRIPTION
**Pull request by jmark**
Title says it all. :)

### CODE REVIEW

File: src/mesh/vtk_helper.jl [hunk 1]
```diff
@@ -33,7 +33,7 @@ Compute the coordinates of a `VTK_LAGRANGE_TRIANGLE` of a triangle or order `ord
 defined by the coordinates of the vertices given in `corner_verts`. `dim` is the
 dimension of the coordinates given. If `skip` is set to true, the coordinates
 of the vertex- and edge-points aren't computed, which can be used to compute
-points of a `VTK_LAGRANGE_WEDGE`
+points of a `VTK_LAGRANGE_TRIANGLE`
```
Comment (jlchan) [inline src/mesh/vtk_helper.jl:36]: Thanks for catching this!

File: src/mesh/vtk_helper.jl [hunk 2]
```diff
@@ -124,6 +124,90 @@ function quad_vtk_order(corner_verts, order, dim, skip = false)
...
+
+    @assert order >= 0 "`order` must be non-negative."
+    
+    coords = copy(corner_verts)
+    if order == 0
+        return coords
+    end
+    
+    # Edges.
+    num_verts_on_edge = order - 1
```
Comment (jlchan) [inline src/mesh/vtk_helper.jl:147]: `verts` here refers to nodes on a given edge, right? I noticed the other routines also use this terminology. Would you mind doing a search-and-replace for the other instances too?
```suggestion
    num_nodes_on_edge = order - 1
```

File: src/mesh/vtk_helper.jl [hunk 3]
```diff
@@ -124,6 +124,90 @@ function quad_vtk_order(corner_verts, order, dim, skip = false)
...
+    edges = SVector(
+      (1,2), (2,3), (4,3), (1,4), # bottom face
+      (5,6), (6,7), (8,7), (5,8), # top face
+      (1,5), (2,6), (4,8), (3,7), # vertical lines
+    )
```
Comment (jlchan) [inline src/mesh/vtk_helper.jl:152]: Is this VTK ordering? If so, can you add a comment noting that?

**Review by jlchan — CHANGES_REQUESTED** [review]
Thanks! I appreciate you adding this. Just have a few minor suggestions.

File: src/mesh/vtk_helper.jl (hunk 2 above)
Comment (jmark) [reply src/mesh/vtk_helper.jl:147]: Done.

File: src/mesh/vtk_helper.jl [hunk 4]
```diff
@@ -124,6 +124,90 @@ function quad_vtk_order(corner_verts, order, dim, skip = false)
...
+
+    # Faces.
+    quad_faces = SVector(
+      (1,4,8,5), # left
+      (2,3,7,6), # right
+      (1,2,6,5), # front
+      (4,3,7,8), # back
+      (1,2,3,4), # bottom
+      (5,6,7,8), # top
+    )
```
[the remark this answers is not in the document]
Comment (jmark) [reply src/mesh/vtk_helper.jl:172]: Done.

File: src/mesh/vtk_helper.jl (hunk 3 above)
Comment (jmark) [reply src/mesh/vtk_helper.jl:152]: Done.

**Review by jlchan — APPROVED** [review]
Thanks!

**Pull request merged**